docs/content/use-cases/nodejs.md

```diff
@@ -5,8 +5,8 @@ weight: 1
 
 ## Installing Alternate NodeJS Versions
 
-The Docksal `cli` comes pre-installed with NodeJS 8.11.3. This is the recommended version for most users. If you require
-a newer version, you have options:
+The Docksal `cli` comes pre-installed with NodeJS 8.11.3. This is the recommended version for most users.  
```

Comment:
10.16 is now the recommended version. Weren’t we talking about updating?

Reply:
Ah 10.15 is currently installed and we didn’t update the docs.

Reply:
Rewrote to say LTS release instead.